README.md

```diff
@@ -17,61 +17,64 @@ This [Snap](https://snapcraft.io/) is part the
+Installation progress can be monitored at all times using `journalctl`.
 
-**Building Magma AGW snap:**
+To install [magma-access-gateway](https://snapcraft.io/magma-access-gateway) execute:
 
-1. Install required software listed above
-2. Checkout [magma-access-gateway-snap](https://github.com/canonical/magma-access-gateway-snap)
-3. Inside checked out repository's main directory execute:<br>
-   `snapcraft`<br>
-   To see what's happening during Snap's building process, `-d` can be used along with above 
-   command.
+```bash
+snap install magma-access-gateway --classic
+```
 
-**Installing Magma AGW snap:**
+Once Snap is installed (it will be indicated by the `magma-access-gateway 1.6.0 installed`
+message printed to the console), start AGW deployment by executing:
+
+```bash
+magma-access-gateway.install
+```
 
-1. Copy snap to AGW host machine.
-2. On AGW host machine, as `root` execute<br>
-   `snap install ${PATH_TO_THE_SNAP_FILE} --dangerous --classic`<br>
-   Since Snap has been built locally, `--dangerous` flag is required to deploy it.
-3. Once Snap is installed (it will be indicated by the `magma-access-gateway 1.6.0 installed`
-   message printed to the console), start AGW deployment by executing:<br>
-   - `magma-access-gateway` - to deploy Magma AGW using DHCP configured SGi interface
-   - `magma-access-gateway --ip-address 1.1.1.1/24 --gw-ip-address 1.1.1.1` - to deploy Magma AGW
-     using statically configured SGi interface
+[magma-access-gateway](https://snapcraft.io/magma-access-gateway) installer exposes various
+deployment configuration parameters. To see the list of currently supported parameters along with
+their descriptions, execute:
 
-   Magma AGW uses **Google (8.8.8.8)** and **OpenDNS (208.67.222.222)** DNS servers by default. 
-   DNS servers can be configured by using `--dns` flag along with `magma-access-gateway` 
-   command. Custom DNS servers should be specified as space separated list of IP addresses.
+```bash
+magma-access-gateway.install --help
+```
 
 **Configuring Magma Access Gateway:**
 
-Magma Access Gateway can be configured by executing:<br>
-`magma-access-gateway.configure --domain <your domain> --root-ca-pem-path <path to Root CA PEM>`
-<br>
+Magma Access Gateway can be configured by executing:
+
+```bash
+magma-access-gateway.configure --domain <your domain> --root-ca-pem-path <path to Root CA PEM>
+```
+
 For more details about Magma Access Gateway configuration, visit 
 [Magma documentation](https://docs.magmacore.org/docs/next/lte/deploy_config_agw).
 
 **Magma Access Gateway post-installation checks**
 
 Once Magma Access Gateway is installed and configured, post-installation checks can be run to 
 make sure everything has been configured correctly.<br>
-Post-installation checks can be executed by issuing:<br>
-`magma-access-gateway.post-install`
+Post-installation checks can be executed by issuing:
+
+```bash
+magma-access-gateway.post-install
+```
 
 ## Contributing
 
```

Comment:
I agree they are similar, but README is automatically displayed on GitHub's main page, so I think it should be there. To avoid duplication, I've updated README to point to How-to guide for the usage part. 
Hope that's fine.